### ISSUE TITLE
[Supporting the platform-Home Page-Navigation]Focus indicator is not visible on all the Sections element like links on all 4 High contrast modes. 

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2604]
**User Experience:**
If a user is not able to see the content properly in any high contrast themes, then this will make it difficult for him to interpret the information properly. This will hinder him from to use the page properly. 

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Pre-Requisite:** Turn on the High Contrast Settings(Settings>Ease of Access>High Contrast)

**Repro Steps:-**
1. Navigate to url:- https://makecode.com/
2. Navigate through all controls (say computing education, about Microsoft makecode, get inspired, resources elements etc)lying on the page.3. Navigate to the Help control lying in the header section on the page and select it.
3. Turn on the High Contrast mode.
4. Verify that focus indicator and all the elements are visible under all 4 HC  modes.

**Actual Result:-**
Focus indicator is not visible on all the Sections element on all 4 High contrast modes. (HC#1,HC#2,HC Black and HC White)

**Expected Result:-**
Focus indicator should be visible on all the Sections element on all 4 High contrast modes. (HC#1,HC#2,HC Black and HC White)

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}

**Suggested fix:** 
a. Ensure web application adheres to the colors of platform provided high contrast themes 
Ensure the web application content colors are updated as per the provided colors for that specific theme 
Ex: Links/Buttons/Text/Selected text have different set of colors for different themes 
b. Ensure text/icons/images are all visible even with high contrast themes on 
Ex1: In some cases background images are used and they will be invisible due to darker background color of selective theme 
Ex2: white colored icons/images are used which will get merged with white background colored high contrast theme 

**Please refer the Video attached.**

[MAS4.3.1_Home Page.zip](https://github.com/Microsoft/pxt/files/1190140/MAS4.3.1_Home.Page.zip)

**Comment by microsoftsam** [comment]
@veler something you can look into? I can show you how to edit the makecode.com page if you know what the fix is here.

**Comment by veler** [comment]
Fix available on [https://makecode.microbit.org/](https://makecode.microbit.org/)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.com/ and issue is partially fixed. Hence reactivating the bug.

**Comment by veler** [comment]
Can you please add more details? What is fixed, what is not?

**Comment by abchatra** [comment]
@v-mesala  @v-amik we need additional information in this bug.

**Comment by v-mesala** [comment]
@abchatra I have added screenshot where issue is fixed and not fixed for controls
![issue is fixed for hands-on computing education](https://user-images.githubusercontent.com/30434417/30368308-68232b52-988e-11e7-91ae-d5270f8ace2d.png)

**Comment by abchatra** [comment]
MakeCode.com is redirect to Microsoft.com/makecode which is MAS compliant and owned by a different team. Closing this bug as not existent anymore.
